src/dom-utils/isTableElement.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+import getComputedStyle from './getComputedStyle';
+
+export default (element: Element) =>
```

Comment:
You need to define the return type
```suggestion
export default (element: Element): boolean =>
```